File: PRAK601_2210817110001_AkhmadRaihanRidha.c
#include <stdio.h>
int main(){
    int j,Baris,i,kolom,Angka;
    scanf("%d  %d", &Baris, &kolom);
    int Matriks[20][20];
    
    for ( i = 0; i < Baris; i++){
        for ( j = 0; j < kolom; j++){
     scanf("%d", &Matriks[i][j]);
            
        }
        
    }
    for ( i = 0; i < Baris; i++){
        for ( j = 0; j < kolom; j++){
            printf("%d ", Matriks[i][j]);
        }
    printf("\n");    
    }
    

}

File: PRAK601_2210817110001_AkhmadRaihanRidha.py
Kolom,baris = map(int,input().split())
Angka= list(map(int,input().split()))
matriks= []
a=0
for i in range (Kolom):
    matriks.append(Angka[a:a+baris])
    a = a + baris
for kolom in matriks:
    for Angka in kolom:
        print(Angka, end=' ')
    print()    

  

